test/configure.test.ts

```diff
@@ -462,13 +462,13 @@ suite("Configure (Add Docker files to Workspace)", function (this: Suite): void
             assertFileContains('docker-compose.debug.yml', '9229:9229');
             assertFileContains('docker-compose.debug.yml', 'image: testoutput');
             assertFileContains('docker-compose.debug.yml', 'NODE_ENV: development');
-            assertFileContains('docker-compose.debug.yml', 'command: node --inspect index.js');
+            assertFileContains('docker-compose.debug.yml', 'command: node --inspect:0.0.0.0:9229 index.js');
```

Comment:
Did the syntax of the node command change, or is this merely reacting to the other changes you've made?

Reply:
The syntax should change in the debug.yml. This is updating the test to account for the new correct value.

The question we need to answer is "What is the use case for NOT passing the correct host and port when "node" isn't found as a command in the `package.json` file? I can't think of one, but I'm sure it was originally done like that for a reason.